README.md

```diff
@@ -27,6 +27,7 @@ This package is currently in the active development.
 
 * [PHP 7.2](http://php.net/releases/7_2_0.php) or greater
 * [Symfony 4.2](https://symfony.com/roadmap/4.2) or [Symfony 3.4](https://symfony.com/roadmap/3.4)
+* [league/oauth2-server (versions >=7.2.0 <7.3.0)](https://packagist.org/packages/league/oauth2-server)
```

Comment:
The constraint in our composer.json is `^7.2` which means >=7.2.0 <8.0 (not < 7.3.0).

Reply:
@X-Coder264 my bad, didn't get my morning coffee yet. Resolved.